rules/cross-platform/data_destruction_meow_attack_detected.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/08/10"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+The Meow attack targets insecurely configured Elasticsearch, MongoDB, and Cassandra databases. The attack destroys the
+contents of the database and replaces (or appends) the word "meow" to the database name.
+"""
+false_positives = [
+    """
+    The word "meow" used in database names will trigger this detection rule. Incident Responders should validate the
+    database is expected and modify this rule as appropriate.
+    """,
+]
+index = ["filebeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Potential Meow Database Attack"
+note = "This detection rule requires the Elasticsearch Filebeat module. This rule will not prevent the Meow attack, it will notify the IR team that an attack has taken place and speed the mean-time-to-detection (MTTD)."
```

Comment:
```suggestion
note = "This detection rule requires the Elasticsearch Filebeat module. This rule will not prevent the Meow attack, it will notify the security team that an attack has taken place and decrease mean-time-to-detection (MTTD)."
```